sw/CMakeLists.txt

```diff
@@ -303,15 +303,27 @@ endif()
 # Set CMAKE flags
 
 # specify the C standard
-set(COMPILER_LINKER_FLAGS "\
-  -march=${CMAKE_SYSTEM_PROCESSOR} \
-  -w -Os -g  -nostdlib  \
-  -ffunction-sections \
-  -DHOST_BUILD \
-  -D${CRT_TYPE} \
-  -D${CRTO} \
-  -DportasmHANDLE_INTERRUPT=vSystemIrqHandler\
-")
+if(NOT ${PROJECT} MATCHES "coremark")
```

Comment:
@JoseCalero , let me know if this is OK with you, then let's merge this PR in as it is now ready

Reply:
this is OK, it can be merged